db/migrate/20230821104043_create_stars_table.rb

```diff
@@ -0,0 +1,15 @@
+class CreateStarsTable < ActiveRecord::Migration[6.1]
+  def change
+    create_table :stars, id: :uuid, default: 'uuid_generate_v4()', null: false do |t|
+      t.uuid :user_id, null: false
+      t.uuid :intervention_id, null: false
+    end
+
+    add_index :stars, :user_id
+    add_index :stars, :intervention_id
+    add_index :stars, %i[user_id intervention_id], unique: true
+
+    add_foreign_key :stars, :users
+    add_foreign_key :stars, :interventions
```

Comment:
Should it be added as default? 
You can use `t.references` 😉

Reply:
I am not sure about this, it is basically a copy of `db/migrate/20200918174450_create_user_interventions.rb`, is this redundant there as well?

Reply:
I understand your way gives you the same result. Personally, I think we can use something which does the job for us in a better and clearer way in smaller lines of the code. I have told you that the approach to doing some things changes over time and you can find both ways in our code.